lib/ansible/modules/network/haproxy.py

```diff
@@ -288,13 +288,14 @@ def get_state_for(self, pxname, svname):
         """
         data = self.execute('show stat', 200, False).lstrip('# ')
         r = csv.DictReader(data.splitlines())
+
         state = tuple(
             map(
                 lambda d: {'status': d['status'], 'weight': d['weight']},
                 filter(lambda d: (pxname is None or d['pxname'] == pxname) and d['svname'] == svname, r)
             )
         )
-        return state or None
+        return list(state) or None
```

Comment:
Why is this needed?  It looks like state should already be a tuple which is a Sequence type?